python/rbac/endpoints/policy/services.py

```diff
@@ -0,0 +1,151 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I think, general idea was to have _services_ as level completely separated from models and from endpoints as well - in `python/rbac/services/` directory for example.